src/app/_theme.scss

```diff
@@ -0,0 +1,56 @@
+// Theme variables
+:root {
+  // Base colours
+  --theme-accent-primary: var(--dim-brand); // Interaction styles (hover, active, focus)
+  --theme-accent-secondary: #68a0b7; // Nested interaction (character menu), armor/weapon stat deltas
+
+  // Generic fills for modal backgrounds (see below)
+  --theme-fill-modal: #000; // Base fill for popups, drawers, dropdowns etc
+  --theme-fill-modal-actions: #161616; // Layered/contextual modals (search, item actions)
+
+  // Buttons
+  --theme-button: rgba(255, 255, 255, 0.2);
```

Comment:
What do you think of having these be things like `--theme-button-bg` to go with `--theme-button-txt`?

Reply:
Sounds good. This feels like a clearer, more future-proof naming convention.

In order to maintain accessible contrast per-theme we may also need new interaction variants (i.e. hovers for the text/bg) — so I think it's probably best to forego actually adding in new txt vars until it's a bit clearer which new vars might to be needed. 

**This raises a broader architectural question**: is it better to establish a naming convention based around the individual components. I think so. 

At the moment i've prefixed them with the style property e.g. `--theme-[background]-item-popup` but it feels like grouping these by component will make it easier to manage themes.

To summarise (since this'll result in a find&replace with a lot of changes) to standardise the variable naming convention as 

```
--theme-[component]-[variant if needed]-[property]
```

Which would result in 

- `--theme-item-popup-bg`
- `--theme-item-popup-actions-bg`
- `--theme-item-feed-bg`

Or hypothetically in future with styles vars such as 

- `--theme-button-bg`
- `--theme-button-txt`
- `--theme-button-hover-bg`
- `--theme-button-hover-txt`
- `--theme-tooltip-bg`
- `--theme-tooltip-txt`

Reply:
I think that makes sense and it's a good convention. Pretty necessary since vars can't be namespaced. Hopefully we solve the duplication by not having separate vars for multiple things that should behave the same - e.g. most things should have the same interaction highlight color.